sdk/physical/file/file_test.go

```diff
@@ -240,3 +240,58 @@ func TestFileBackend(t *testing.T) {
 
 	physical.ExerciseBackend_ListPrefix(t, b)
 }
+
+func TestFileBackendCreateTempKey(t *testing.T) {
+	dir, err := ioutil.TempDir("", "vault")
+	if err != nil {
+		t.Fatalf("err: %s", err)
+	}
+	defer os.RemoveAll(dir)
```

Comment:
This is super nitpicky, but if you use [t.TempDir()](https://pkg.go.dev/testing@go1.22.0#T.TempDir) here, you can avoid error handling and cleanup, as all of that is automatic.